cli/azd/pkg/azdo/repository.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azdo
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/input"
+	"github.com/microsoft/azure-devops-go-api/azuredevops"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/git"
+)
+
+// create a new repository in the current project
+func CreateRepository(ctx context.Context, projectId string, repoName string, connection *azuredevops.Connection) (*git.GitRepository, error) {
+	gitClient, err := git.NewClient(ctx, connection)
+	if err != nil {
+		return nil, err
+	}
+
+	gitRepositoryCreateOptions := git.GitRepositoryCreateOptions{
+		Name: &repoName,
+	}
+
+	createRepositoryArgs := git.CreateRepositoryArgs{
+		Project:               &projectId,
+		GitRepositoryToCreate: &gitRepositoryCreateOptions,
+	}
+	repo, err := gitClient.CreateRepository(ctx, createRepositoryArgs)
+	if err != nil {
+		return nil, err
+	}
+	return repo, nil
+}
+
+// returns a default repo from a newly created AzDo project.
+// this relies on the fact that new projects automatically get a repo named the same as the project
+func GetAzDoDefaultGitRepositoriesInProject(ctx context.Context, projectName string, connection *azuredevops.Connection) (*git.GitRepository, error) {
```

Comment:
Same regarding `Azdo` in the function names.
```suggestion
func GetDefaultGitRepositoriesInProject(ctx context.Context, projectName string, connection *azuredevops.Connection) (*git.GitRepository, error) {
```

Reply:
Same as above, this was from before I refactored the code into an azdo package. I will fix.